recipes-kernel/linux/linux-sunxi_3.4.bb

```diff
@@ -3,10 +3,11 @@ require linux.inc
 DESCRIPTION = "Linux kernel for Allwinner a10/a20 processors"
 
 COMPATIBLE_MACHINE = "(sun4i|sun5i|sun7i)"
+DEPENDS = "sunxi-board-fex"
 
-PV = "3.4.103"
+PV = "3.4.104"
 PR = "r1"
```

Comment:
It would be better to put it in it´s own pull request, but the change is not that big so I used the same commit.